js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -274,6 +274,8 @@ const KeyboardNavigationController = core.ViewController.inherit({
 
     // #region Key_Handlers
     _keyDownHandler: function(e) {
+        const isCommandKeyPressed = e.ctrl || e.metaKey;
```

Comment:
Why isCommandKeyPressed utility method is not used in this file?

Reply:
I think we should use isCommandKeyPressed(originalEvent).
In the keyboardProcessor:
```
ctrl: e.ctrlKey,
metaKey: e.metaKey,
originalEvent: e
```